src/styles/stars.css

```diff
@@ -0,0 +1,17 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background: url(../images/star.svg) center no-repeat;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--4 .stars__star:not(:last-child) {
```

Comment:
according to the task, there should be logic for all options, if you change `stars--4` modifier to` stars--2 `or other number of selected stars should change correctly, so you need to add `.stars--1`, `.stars--2`... etc.